plugin/pkg/admission/security/podsecuritypolicy/admission_test.go

```diff
@@ -344,6 +345,11 @@ func TestAdmitPreferNonmutating(t *testing.T) {
 	gcChangedPod.OwnerReferences = []metav1.OwnerReference{{Kind: "Foo", Name: "bar"}}
 	gcChangedPod.Finalizers = []string{"foo"}
 
+	podWithAnnotation := unprivilegedRunAsAnyPod.DeepCopy()
+	podWithAnnotation.ObjectMeta.Annotations = map[string]string{
+		psputil.ValidatedPSPAnnotation: mutating2.Name,
```

Comment:
add a comment pls that "mutating2" is lexicographically behind "mutating1", so "mutating1" should be chosen because it's the canonical PSP order.

Reply:
Done.